pkg/helidonapp/helidonapp_test.go

```diff
@@ -37,6 +37,46 @@ func TestCreateHelidonAppCR(t *testing.T) {
 	assert.Equal(t, int32(targetPort), helidonApp.Spec.TargetPort, "Expected TargetPort")
 }
 
+func TestCreateHelidonAppWithCoherenceCluster(t *testing.T) {
+	mcName, namespace, appName := "myCluster", "myNs", "myHelidonApp"
```

Comment:
Might be good to use the GIVEN, WHEN, THEN doc for these tests.

Reply:
godocs added